README.md

```diff
@@ -40,6 +40,8 @@ As the Prisoner Search is a complex API with multiple dependencies, we've opted
 
 **Warning:** the mocked API currently only covers the Find journey and searching for a person before creating a referral. The rest of the Refer journey will not work with a mocked API at present.
 
+Mocked referral summaries and referrals are mocked to have been referred by `ACP_POM_USER`.  To view these referrals with the `status` set to `referral_started`, you must be logged in as `ACP_POM_USER`.
```

Comment:
There's a double space between the sentences but I don't think Markdown cares about that